cppcache/integration-test/CacheHelper.cpp

```diff
@@ -134,7 +134,7 @@ CacheHelper::CacheHelper(const std::shared_ptr<Properties>& configPtr,
   auto cacheFactory = CacheFactory(pp);
   cachePtr = std::make_shared<Cache>(cacheFactory.create());
 
-  auto poolFactory = cachePtr->getPoolManager().createFactory();
+  auto poolFactory = std::make_shared<PoolFactory>(cachePtr->getPoolManager().createFactory());
```

Comment:
Is the pool factory assigned to a member variable later? If not then we don’t need the shared_ptr.

Reply:
It is passed around later on through method calls. We had kept it as shared_ptr to minimize our code changes.  But your requested change isn't that bad so it has been updated in next revision.